pkg/oam/util/helper_test.go

```diff
@@ -715,11 +716,37 @@ var _ = Describe("Test unstructured related helper utils", func() {
 				}},
 				exp: "simplerollouttraits.extend.oam.dev",
 			},
+			"trait": {
+				u: &unstructured.Unstructured{Object: map[string]interface{}{
+					"apiVersion": "extend.oam.dev/v1alpha2",
+					"kind":       "SimpleRolloutTrait",
+					"metadata": map[string]interface{}{
```

Comment:
it will be good to add a name here that is different from the label so we know which is the truth